src/CodeGen_OpenGL_Dev.cpp

```diff
@@ -250,6 +250,14 @@ void CodeGen_GLSL::visit(const FloatImm *op) {
     id = oss.str();
 }
 
+void CodeGen_GLSL::visit(const IntImm *op) {
+    print_assignment(op->type, print_type(op->type) + "(" + std::to_string(op->value) + ")");
```

Comment:
Oh, if this is incorrect in C, that would be a problem for OpenCL C. If there's any doubt, let's just leave it alone, it's a really small change. My suggestion was an "if all else equal..." kind of thing.